tests/json_test/json_test.cpp

```diff
@@ -9464,6 +9464,26 @@ suite shark_variant = [] {
    };
 };
 
+suite string_view_value_tests = [] {
+   "string_view value"_test = [] {
+      std::string_view value = "";
+      expect(not glz::read_json(value, R"("hello")"));
+      expect(value == "hello");
+   };
+};
+
+suite array_char_tests = [] {
+   "array<char... value"_test = [] {
+      std::array<char, 8> value{};
+      expect(not glz::read_json(value, R"("hello")"));
```

Comment:
Open question : should a `std::array<char, N>` considered as a string with a **maximum** capacity of `N` or a string of **exactly** `N` ? For me, it would make more sense to only accept exact lengths of `N`. As an array is not a vector, it always have a fixed length, and thus it should return an error if the value length is under `N`.